### PR TITLE
Use dotnetcli in DiagnosticScenarios/readme.md

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes dotnet/docs#16283.

I've also updated the "name" in the metadata because the h1 seems to be ignored, and the "name" is used as the h1.

image

### CODE REVIEW

File: core/diagnostics/DiagnosticScenarios/readme.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ languages:
 products:
 - dotnet-core
 page_type: sample
-name: "Diagnostic Scenarios"
+name: "DiagnosticScenarios sample debug target"
 urlFragment: "diagnostic-scenarios"
 description: "A .NET Core sample with methods that trigger undesirable behaviors to diagnose."
 ---
 # DiagnosticScenarios sample debug target
```
Comment (Youssef1313) [inline core/diagnostics/DiagnosticScenarios/readme.md]: This heading isn't rendered as shown in the screenshot. so, I updated the "name" in the metadata section.

Comment (Youssef1313) [reply core/diagnostics/DiagnosticScenarios/readme.md]: @mairaw, @BillWagner, Should the h1 be completely deleted as they're not rendered at all ?

**Comment by mairaw** [comment]
Thanks @Youssef1313. I think this one should be closed in favor of https://github.com/dotnet/samples/pull/1857 that got merged. Or would there be anything left after merge conflicts are resolved?

**Comment by Youssef1313** [comment]
The only thing I see can be done to the readme is to use dotnetcli identifier. Should I resolve the conflicts and make this small fix or just close and leave as-is ?

**Comment by sdmaclea** [comment]
> use dotnetcli identifier

I missed that....

File: core/diagnostics/DiagnosticScenarios/readme.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ The sample debug target is a simple `webapi` application. The sample triggers un
 
 After downloading the source, you can easily run the webapi using:
 
-```console
+```dotnetcli
 dotnet build
 dotnet run
 ```
```
Comment (mairaw) [inline core/diagnostics/DiagnosticScenarios/readme.md]: Thanks @Youssef1313! The links on lines 30, 36, 42, 48 could use a code block with the http identifier as well. Then people should get a copy link like [here](https://docs.microsoft.com/en-us/dotnet/framework/data/wcf/querying-the-data-service-wcf-data-services#linq-queries)

File: core/diagnostics/DiagnosticScenarios/readme.md (hunk 1 above)
Comment (mairaw) [reply core/diagnostics/DiagnosticScenarios/readme.md]: I'd have to investigate. I haven't looked at the samples guidance yet.

File: core/diagnostics/DiagnosticScenarios/readme.md (hunk 2 above)
Comment (Youssef1313) [reply core/diagnostics/DiagnosticScenarios/readme.md]: @mairaw done.

**Pull request merged**